snomed/com.b2international.snowowl.snomed.reasoner/src/com/b2international/snowowl/snomed/reasoner/request/SaveJobRequest.java

```diff
@@ -200,26 +196,42 @@ private Boolean persistChanges(final BranchContext context, final IProgressMonit
 		classificationTracker.classificationSaving(classificationId);
 
 		final SubMonitor subMonitor = SubMonitor.convert(monitor, "Persisting changes", 6);
-		final BulkRequestBuilder<TransactionContext> bulkRequestBuilder = BulkRequest.create();
+		final List<RequestBuilder<TransactionContext, ?>> bulkRequestBuilder = Lists.newArrayList();
```

Comment:
You can make this change a lot smaller by skipping all `List<RequestBuilder<TransactionContext, ?>>` changes and `(RequestBuilder<TransactionContext, ?>)` casts below, if you keep the `BulkRequestBuilder` type here, then call `build` on it and ask for its request list via `BulkRequest#getRequests` for the partitioning step.